src/components/Modals/NewMessageModal.jsx

```diff
@@ -131,10 +132,10 @@ const NewMessageModal = ({ showModal, setShowModal, oldMessage = '', toField = '
           display: 'flex',
           flexDirection: 'column',
           justifyContent: 'center',
-          alignItems: 'center',
-          padding: '20px',
-          minWidth: '50%',
-          minHeight: '90%'
+          alignItems: 'center'
+          // padding: '20px',
+          // minWidth: '50%',
+          // minHeight: '90%'
```

Comment:
Think we could have these removed if unused. Think if we do use them, we could have them set in `ModalBase` where we could play with the padding and spacing to get a good fit.